### ISSUE TITLE
Error Feedback Screen (ícone x animação)

### ISSUE THREAD

**Issue by annefranca** [Telefonica/mistica-design#1935]
### Describe the bug

O ícone e a animação aplicada no Error Feedback Screen estão diferentes.

**Figma:** https://www.figma.com/design/JkA7OefLqga6XyfiqLq1dY/Vivo-Extended?m=auto&node-id=22746-60212&t=G3COyYJdBbM8FBvK-1
 
**Playroom:** https://mistica-web.vercel.app/playroom/#?code=N4Igxg9gJgpiBcIA8BRATmiaBiMZQCMBDMAawGUw08A7AHRoAJGAXASxYBsYBeOkchAC2MVh278GzWAGcqbAA7sINPgOGjZ8pWxWsYADxaSmjBWjZCiaAJ4AhAK4sWKnsCSPnKgAoWrtxh9qGRk3AAoASkYeAD5GYABfBJiAQTBlGiQAek8XGl9LaxsYhIYsmIYQABoQFgALGBEZBABtEAA1NgA3CAB9ADkYAHcQAF0aobYoeub4FoBmADYABlGEoA

### What libraries are you seeing the problem on?

Mobile Library, Desktop Library, Vivo Library

### Steps to reproduce

N/A

### Expected behavior

Ter a mesma visualização entre o ícone aplicado no Figma e a animação feita em código.

**Comment by yceballost** [comment]
will be released in 16.0.0

**Comment by annefranca** [comment]
@yceballost pode me confirmar se esse ajuste foi realmente feito na release 16.0.0?

**Comment by yceballost** [comment]
@annefranca sí, se cambió en Figma para que fuese lo más parecido a código, sigues encontrando mucha diferencia?

**Comment by annefranca** [comment]
@yceballost está com uma diferença na espessura e aparentemente a resolução do Error está um pouco ruim.

<img width="653" alt="Image" src="https://github.com/user-attachments/assets/d0f2387d-101a-4a06-8866-05cb97163713" />
